lib/spack/env/cc

```diff
@@ -173,20 +213,37 @@ preextend() {
     unset IFS
 }
 
-# system_dir PATH
+# contains str listval [sep=:] [strip=/]
+# test whether a path is a system directory
+contains() {
+    sep="${3:-:}"
+    # SPACK_SYSTEM_DIRS is colon-separated
```

Comment:
This comment seems misplaced too

Reply:
This one is not actually misplaced, but doesn't make as much sense anymore.  It's kind-of meant to explain the `:` default, will at least reword and possibly remove.